site/content/docs/main/backup-reference.md

```diff
@@ -74,7 +76,23 @@ Please do notice there is also side effect that may not be expected. Because sch
 
 If there is possibility the schedule will be disable to not create backup anymore, and the created backups are still useful. Please do not enable this option. For detail, please reference to [Backups created by a schedule with useOwnerReferenceInBackup set do not get synced properly](https://github.com/vmware-tanzu/velero/issues/4093).
 
+#### Cannot support backup data immutability
+Starting from 1.11, Velero's backups may not work as expected when the target object storage has some kind of an "immutability" option configured. These options are known by different names (see links below for some examples). The main reason is that Velero first saves the state of a backup as Finalizing and then checks whether there are any async operations in progress. If there are, it needs to wait for all of them to be finished before moving the backup state to Complete. If there are no async operations, the state is moved to Complete right away. In either case, Velero needs to modify the metadata in object storage and that will not be possible if some kind of immutability is configured on the object storage.
+
+Please also notice, before 1.11, even Velero can work with the target object storage with an "immutability" configuration, Velero didn't provide any "immutability" options. It can work because there is no limitation, but no official support in the data "immutability" function too.
```

Comment:
This paragraph is a bit confusing to me. Do you mean to say that Velero pre-1.11 will work with immutability option though there will be some issues (such as old versions not being deleted)?

Reply:
No. I mean to say that cannot supporting immutability is a new introducing issue, but Velero doesn't provide any support on this before.

Reply:
OK. In that case, I would suggest this:

"Even with versions prior to 1.11, there was no explicit support in Velero to work with object storage that has  "immutability" configuration. As a result, you may see some problems even though backups seem to work (e.g. versions objects not being deleted when backup is deleted)."

Reply:
Thanks a lot for the rephrasing.